### PR TITLE
adding feature edit profile information.

### PR DESCRIPTION
**Pull request by mohammed0xff**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/users/maryanchuk04/projects/3)

## Code reviewers

- [ ] @maryanchuk04 
- [ ] @Serhiy-dotcom 

### Second Level Review

- [ ] @Vova-Romaniuk 

## Summary of issue

Write simple api for change data of user.
## Summary of change

Create new abstract service. Create api controller with PUT method.
Write new DTO for user.

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: StudCity.Backend/StudCity.API/Controllers/UserController.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        _userService = userService;
+    }
+
+    /// <summary>
+    /// Update user information.
+    /// </summary>
+    /// <param name="user">Model that contains user's data.</param>
+    /// <response code ='204'>When informations updated successfully.</response>
+    [Authorize]
+    [HttpPut("update-info")]
```
Comment (maryanchuk04) [inline StudCity.Backend/StudCity.API/Controllers/UserController.cs:30]: While we have an empty API controller, we can write a real RESTful controller.
That is: 


Method | Route | Description
-- | -- | --
GET | /user | get something...
POST | /user | do something...
PUT |  /user | Update user
DELETE | /user/{id} | delete user...

**Review by maryanchuk04 — CHANGES_REQUESTED** [review]
Fix small mistakes, and in general everything is cool
it is only necessary to implement this service, but I will take care of it. Thank you!

File: StudCity.Backend/StudCity.API/Controllers/UserController.cs (hunk 1 above)
Comment (mohammed0xff) [reply StudCity.Backend/StudCity.API/Controllers/UserController.cs:30]: i want to extend this controller and take the next step in implementing that service..
however i'm waiting for little bit of planning and guidance from you 
thanks a lot, wish you a great day 😀

**Comment by mohammed0xff** [comment]
Hello Maks, I think this project is super cool and I wonder if I can be a
part of it ..
i can receive some tasks and my code be reviewed by you and other
contributors
I'm learning a lot from the code design to things I didn't know how it's
really done.
so if that is possible please let me know.
Thanks for your time.

**Pull request closed without merging**